doc/api/cli.md

```diff
@@ -1669,6 +1669,24 @@ Be aware that unless the child environment is explicitly set, this environment
 variable will be inherited by any child processes, and if they use OpenSSL, it
 may cause them to trust the same CAs as node.
 
+### `TZ`
+<!-- YAML
+added: REPLACEME
```

Comment:
```suggestion
added: v0.0.1
changes:
  - version:
     - REPLACEME
    pr-url: https://github.com/nodejs/node/pull/38642
    description:
      Changing the TZ variable using process.env.TZ = changes the timezone
      on Windows as well.
  - version:
     - v13.0.0
    pr-url: https://github.com/nodejs/node/pull/20026
    description:
      Changing the TZ variable using process.env.TZ = changes the timezone
      on POSIX systems.
```

(I could not actually verify that this worked in v0.0.1, because old Node.js versions don’t even compile with a recent gcc anymore, but the V8 tests for that version already had tests mentioning the env var :))